feature-group-definitions/compute-pressure.yml

```
@@ -0,0 +1,6 @@
+name: Compute Pressure API
+description: A JavaScript API that enables you to observe the pressure of system resources such as the CPU.
```

Trying to fit this into a "The X does Y" wording. Since it's CPU-only, how about:

```suggestion
description: The `PressureObserver` API monitors the CPU pressure of the system, allowing you to reduce pressure when needed. Also known as the Compute Pressure API.
```

---

"Pressure" is a novel term that only means something if you already read the docs. Maybe we avoid the term in the description. Something like:

```suggestion
description: The `PressureObserver` API monitors CPU load, allowing you to adjust workloads in response to available computing resources. Also known as the Compute Pressure API.
```

This specifically names CPU, which is true here and now but might not always be in the future. I guess it's easier to write about what is than what might be.